content/blog/2023-02-09-ethereum-classic-course-8-ethereum-classics-monetary-policy/index.md

```diff
@@ -0,0 +1,153 @@
+
+To this, the founding team allocated to themselves and the Ethereum Foundation another 12,000,000 ETC/ETH.
+
+So, this brought the total pre-mine issuance of ETC/ETH to 72,009,990, and this forms part of the current supply of ETC and ETH in their respective networks.
+
+## 2. Eras
+
+![Each era lasts 5,000,000 blocks.](./3.png)
+
+As mentioned before, starting on block 5,000,000, ETC changed its monetary policy of unlimited supply to a capped supply model as Bitcoin’s.
+
+The schedule followed was that every 5,000,000 blocks, or 28 months, ETC would discount by 20% the payment per block to miners. In this way, it would resemble Bitcoin’s schedule, which discounts the payment to miners by 50%, but every 48 months.
+
+This means that in every era, miners would get paid 20% less as follows:
+
+- Era 1, from block 1 to 5,000,000: 5 ETC
+- Era 2, from block 5,000,001 to 10,000,000: 4 ETC
+- Era 3, from block 10,000,001 to 15,000,000: 3.20 ETC
+- Era 4, from block 15,000,001 to 20,000,000: 2.56 ETC (current era)...
+
+...and so on until block rewards will diminish completely by approximately the year 2140.
+
+## 3. Block Rewards
+
+![Block rewards started at 5 and are discounted from there.](./4.png)
+
+Block rewards are the method of issuing the money fairly and of compensating miners to secure the network by producing blocks.
+
+It is described as “fair” because the coins are being issued and paid to people and entities who actually perform work for the network. Miners secure the network because their work imposes an enormous cost for any bad actors to tamper with the blockchain.
+
+Additionally, the fact that the currency is issued only after performing significant work makes it sound money as it is unforgeably costly and scarce.
+
+Block frequency and block rewards are the key components of the monetary policy because they determine the timing of the issuance, thus inflation rate and stock-to-flow ratio.
+
+For example, as the ETC blocks are produced every 15 seconds on average, and the current block rewards are 2.56 ETC, then, in this era, the rate of currency issuance is 5,382,144 per year. And, this rate of inflation will be reduced by 20% every 28 months.
+
+## 4. Uncle Blocks
+
+![There are, on average, 270,000 uncle blocks per era.](./5.png)
+
+As Ethereum Classic produces blocks every 15 seconds, instead of every ten minutes as Bitcoin, there are instances in which more than one valid block is produced by miners at any given time. However, only one block per round may be permitted, so to prevent any disincentive that miners may have for producing extra blocks in vain, there is a payment for these stale blocks, also called “uncle blocks”.
+
+Since inception, there has been a rate of production of uncle blocks of more or less 5.4%, which is approximately 270,000 uncle blocks per era.
```

Comment:
Line 898: Editorial rewrite
```suggestion
Since inception, uncle blocks are produced at the rate of more or less 5.4% of total blocks which equals approximately 270,000 uncle blocks per era.
```